src/en/rizzcomic/src/eu/kanade/tachiyomi/extension/en/rizzcomic/RealmOasis.kt

```diff
@@ -52,15 +58,24 @@ class RizzComic : MangaThemesiaAlt(
             .build()
     }
 
-    override val versionId = 2
-
-    override val slugRegex = Regex("""^(r\d+-)""")
-
-    // don't allow disabling random part setting
-    override fun setupPreferenceScreen(screen: PreferenceScreen) = Unit
-
-    override val listUrl = mangaUrlDirectory
-    override val listSelector = "div.bsx a"
+    override val versionId = 3
+
+    private val preferences =
+        Injekt.get<Application>().getSharedPreferences("source_$id", 0x0000)
+
+    private val mangaPath by lazy {
+        client.newCall(GET(baseUrl, headers))
+            .execute().asJsoup()
+            .selectFirst(".listupd a")!!
+            .absUrl("href")
+            .toHttpUrl()
+            .pathSegments[0]
+            .also {
+                preferences.edit()
+                    .putString("manga_path", it)
```

Comment:
Missed this the last time. Should put into a const String, as typos can lead to weird bugs without any hint to the cause.